### PR TITLE
restore-pod.yaml: fix temporary restore directory name

### PR DESCRIPTION
**Pull request by alaypatel07**

### CODE REVIEW

File: bindata/etcd/restore-pod.yaml [hunk 1]
```diff
@@ -48,15 +48,18 @@ spec:
         fi
 
         UUID=$(uuidgen)
+        UUID=${UUID:0:10}
```
Comment (hexfusion) [inline bindata/etcd/restore-pod.yaml]: just use the whole ID or another unique string

Comment (alaypatel07) [reply bindata/etcd/restore-pod.yaml]: done

**Comment by hexfusion** [comment]
/close

continued on #257

**Pull request closed without merging**